cmd/modelcmd/base.go

```diff
@@ -336,6 +336,15 @@ func (c *CommandBase) ModelUUIDs(store jujuclient.ClientStore, controllerName st
 	return result, nil
 }
 
+// ControllerUUID returns the controller UUID for specified controller name.
+func (c *CommandBase) ControllerUUID(store jujuclient.ClientStore, controllerName string) (string, error) {
```

Comment:
this should probably go on ControllerCommand if possible

Reply:
not sure why `add-k8s` is an baseCMD rather than controlerCMD.
should we change?